app/src/main/java/com/example/footprnt/Profile/EditPost.java

```diff
@@ -120,6 +100,50 @@ public void done(ParseException e) {
             }
         });
 
+
+        mBtnSave.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                post.setTitle(mEtTitle.getText().toString());
+                post.setDescription(mEtDescription.getText().toString());
+                post.getUser().saveInBackground(new SaveCallback() {
+                    @Override
+                    public void done(ParseException e) {
+                        Intent it = new Intent();
+                        it.putExtra("position", position);
+                        setResult(301, it);
```

Comment:
301 should be a constant so it never gets out of sync with the onActivityResult that's listening to it.